### PR TITLE
#319 Added links to Resources page

### PR DESCRIPTION
**Pull request by webuxr**
Adding links mentioned in #319 to Resources YAML.

### CODE REVIEW

File: _data/resources.yml [hunk 1]
```diff
@@ -217,4 +223,8 @@
       title: "Sass a11y"
     - url: http://tenon.io/
       title: "Tenon.io Accessibility Testing API"
+    - url: http://jxnblk.com/colorable/demos/text/
```
Comment (davatron5000) [inline _data/resources.yml]: Are these added twice? Let's try not duplicating and maybe just choose category.

Comment (webuxr) [inline _data/resources.yml]: @davatron5000 Yes, they are repeated as they're applicable to both "Online Tools" and "Development Tools". I'd be OK with removing these from Dev Tools.

To summarize:

- Add Frend to "HTML Related" category (remove it from any others)
- Add A11y Colors and Colorable to "Online Tools" category (remove them from any others)

Thoughts?

**Comment by webuxr** [comment]
Thank you @davatron5000

**Comment by webuxr** [comment]
Thank you @davatron5000

**Pull request merged**